packages/api-v2/test/student/student.e2e-spec.ts

```diff
@@ -0,0 +1,125 @@
+import { INestApplication } from "@nestjs/common";
+import { Test, TestingModule } from "@nestjs/testing";
+import { Plan } from "../../src/plan/entities/plan.entity";
+import { Student } from "../../src/student/entities/student.entity";
+import * as request from "supertest";
+import { Connection } from "typeorm";
+import { AppModule } from "../../src/app.module";
+
+const testUser = {
```

Comment:
Same as what Alex indicated below, I think we can create a new file with all the test users/plans, then simply import all the users/plans we need